.github/workflows/msys2_build.yml

```diff
@@ -0,0 +1,106 @@
+name: Windows MSYS2 build
+on: [push, pull_request]
+
+env:
+  # Customize the env if
+  # Disable since mtl patched dpdk pcapng and this job use prebuilt pkg
+  MTL_BUILD_DISABLE_PCAPNG: true
```

Comment:
Should remove `MTL_BUILD_DISABLE_PCAPNG` as DPDK is built from code in this job.

Reply:
Sure, thanks.